modules/network-observability-flowcollector-kafka-config.adoc

```diff
@@ -5,14 +5,29 @@
 :_content-type: PROCEDURE
 [id="network-observability-flowcollector-kafka-config_{context}"]
 = Configuring the Flow Collector resource with Kafka
-You can configure the `FlowCollector` resource to use Kafka. A Kafka instance needs to be running, and a Kafka topic dedicated to {product-title} Network Observability must be created in that instance. For more information, refer to your Kafka documentation, such as link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html/using_amq_streams_on_openshift/using-the-topic-operator-str[Kafka documentation with AMQ Streams].
+You can configure the `FlowCollector` resource to use Kafka for storage. A Kafka instance needs to be running, and a Kafka topic dedicated to {product-title} Network Observability must be created in that instance. For more information, refer to your Kafka documentation, such as link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html/using_amq_streams_on_openshift/using-the-topic-operator-str[Kafka documentation with AMQ Streams].
```

Comment:
I don't think "for storage" is relevant here. Kakfa is used to push data forward but not directly to store.

Reply:
+1; what about something like that ? (taken from their own description)

```suggestion
You can configure the `FlowCollector` resource to use Kafka for high-throughput and low-latency data feeds. A Kafka instance needs to be running, and a Kafka topic dedicated to {product-title} Network Observability must be created in that instance. For more information, refer to your Kafka documentation, such as link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html/using_amq_streams_on_openshift/using-the-topic-operator-str[Kafka documentation with AMQ Streams].
```

Reply:
Ok, thanks for this discussion. In my talks with Amogh about the different choices customers have there's Use Kafka as an Exporter, and then we just have Install Kafka and Configure Kafka with the Flow Collector, but these mentions don't explain the utility of this implementation of Kafka. I'll take your suggestion, Julien. I think this is what I was looking for and trying to clarify.

Reply:
Addressed.